src/js/ui/eac/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { header as headerWidget , menuButton} from '../shared/common';
+import redraw from '../../utils/redraw';
+import * as helper from '../helper';
+import formWidgets from '../shared/form';
+import layout from '../layout';
+import i18n, { loadFromSettings, getAvailableLanguages } from '../../i18n';
+import settings from '../../settings';
+import * as m from 'mithril';
+import * as xhr from '../../xhr';
+import * as stream from 'mithril/stream';
+import * as utils from '../../utils';
+import router from '../../router';
+import { Pos } from '../../lichess/interfaces';
+export default {
+  oncreate: helper.viewSlideIn,
+
+  oninit: function() {
+    this.positions = stream([]);
+    this.endgamesPositions = stream([]);
+    this.position = '';
+
+    utils.loadLocalJsonFile('data/endgames.json')
+      .then(data => {
+        this.endgamesPositions(data);
+        redraw();
+      });
+  },
+
+  view: function() {
+    const ctrl = this;
+
+
+    const header = headerWidget.bind(undefined, null, goToGame());
+    function renderPosition(pos: Pos) {
+      return (
+        <li className="list_item">
+          {formWidgets.renderRadio(pos.name, 'position', pos.fen,
+            ctrl.position === pos.fen,
+            e => {
+              ctrl.position = pos.fen;
+            }
+          )}
+        </li>
+      );
+    }
+
+    function renderBody() {
+      return (
+        <ul className="native_scroller page settings_list radio_list">
+          {ctrl.endgamesPositions().map((p: Pos) => renderPosition(p))}
+        </ul>
+      );
+    }
+
+    function start() {
+      settings.gameSetup.ai.level('8');
+      settings.gameSetup.ai.variant('3')
+      settings.gameSetup.ai.color('white');
+      function startAIGame() {
+        return xhr.newAiGame(ctrl.position)
```

Comment:
there is an option missing ? this one settings.gameSetup.ai.availableVariants[1] ?